frontend/app/routes/api.switch-language.ts

```diff
@@ -2,13 +2,14 @@ import { type ActionFunctionArgs, json } from '@remix-run/node';
 
 import { z } from 'zod';
 
+import { PREFFERRED_LANGUAGES } from '~/utils/constants';
 import { createLangCookie } from '~/utils/locale-utils.server';
 import { getLogger } from '~/utils/logging.server';
 
 const logger = getLogger('routes/api.switch-language');
 
 const switchLanguageDataSchema = z.object({
-  language: z.enum(['en', 'fr']),
+  language: z.enum(PREFFERRED_LANGUAGES),
```

Comment:
**Thought/Nitpick:** I feel like the preferred languages array shouldn't be tied into the language switcher as they are two independent requirements. If down the line we need to add another preferred language like "Both/Bilingual", we may not want that option for the language switcher.

Reply:
Just saw @sebastien-comeau with the same comment so I will resolve this.